npm/webpack-dev-server/src/createWebpackDevServer.ts

```diff
@@ -68,7 +68,7 @@ function webpackDevServer4 (
   const { devServerConfig: { cypressConfig: { devServerPublicPathRoute } } } = config
   const WebpackDevServer = config.sourceWebpackModulesResult.webpackDevServer.module
   const webpackDevServerConfig = {
-    host: 'localhost',
+    host: '127.0.0.1',
```

Comment:
In general, we should never use `'localhost'` in the App due to issues like this. We've even had users with an incorrect `localhost` entry in `/etc/hosts`, which causes mind boggling issues.

Reply:
I was absolutely baffled as to what was going on until I stumbled upon the issues I listed above.